av1an-core/src/progress_bar.rs

```diff
@@ -144,23 +144,23 @@ pub fn reset_mp_bar_at(pos: u64) {
   }
 }
 
-pub fn init_multi_progress_bar(len: u64, workers: usize) {
+pub fn init_multi_progress_bar(len: u64, workers: usize, total_chunks: usize) {
   MULTI_PROGRESS_BAR.get_or_init(|| {
     let mpb = MultiProgress::new();
 
     let mut pbs = Vec::new();
 
-    let digits = printable_base10_digits(workers) as usize;
+    let digits = printable_base10_digits(total_chunks) as usize;
```

Comment:
Oh one very small thing actually. This should be `total_chunks.saturating_sub(1)` (in case of overflow I guess?) since we display the index.

Reply:
You are technically correct. The best kind of correct. (the outcome of not fixing it is if there are exactly 10 or 100 or 1000 chunks, there's an extra space. I'll still open another PR to fix it)

Reply:
Ah, you don't need to worry about it. I already pushed a commit to this PR that fixes it :)